documentation/docs/05-misc/04-v4-migration-guide.md

```diff
@@ -0,0 +1,57 @@
+
+## Stricter types for Svelte functions
+
+There are now stricter types for `createEventDispatcher`, `Action`, `ActionReturn`, and `onMount`:
+
+- `createEventDispatcher` now supports specifying that a payload is optional, required, or non-existent, and the call sites are checked accordingly ([#7224](https://github.com/sveltejs/svelte/issues/7224))
+- `Action` and `ActionReturn` have a default parameter type of `never` now, which means you need to type the generic if you want to specify that this action receives a parameter. The migration script will migrate this automatically ([#7442](https://github.com/sveltejs/svelte/pull/7442))
+- `onMount` now shows a type error if you return a function asynchronously from it, because this is likely a bug in your code where you expect the callback to be called on destroy, which it will only do for synchronously returned functions ([#8136](https://github.com/sveltejs/svelte/issues/8136))
+
+## Custom Elements with Svelte
+
+The creation of custom elements with Svelte has been overhauled and significantly improved. The `tag` option is deprecated in favor of the new `customElement` option. The migration script will adjust your code automatically. The update timing of properties has changed slightly as well. ([#8457](https://github.com/sveltejs/svelte/issues/8457))
+
+## SvelteComponentTyped is deprecated
+
+`SvelteComponentTyped` is deprecated, as `SvelteComponent` now has all its typing capabilities. Replace all instances of `SvelteComponentTyped` with `SvelteComponent`. If you have used `SvelteComponent` as the component instance type previously, you may see a somewhat opaque type error now, which is solved by changing `: typeof SvelteComponent` to `: typeof SvelteComponent<any>` (more info in the linked PR). The migration script will do both automatically for you. ([#8512](https://github.com/sveltejs/svelte/issues/8512))
+
+## Transitions are local by default
+
+Transitions are now local by default to prevent confusion around page navigations. To make them global, add the `|global` modifier. The migration script will do this automatically for you. ([#6686](https://github.com/sveltejs/svelte/issues/6686))
+
+## Default slot bindings
+
+Default slot bindings are no longer exposed to named slots and vice versa. ([#6049](https://github.com/sveltejs/svelte/issues/6049))
+
+## Preprocessors
+
+The order in which preprocessors are applied has changed. Now, preprocessors are executed in order, and within one group, the order is markup, script, style. Each preprocessor must also have a name. ([#8618](https://github.com/sveltejs/svelte/issues/8618))
+
+## Other breaking changes
+
+- the `inert` attribute is now applied to outroing elements to make them invisible to assistive technology and prevent interaction. ([#8627](https://github.com/sveltejs/svelte/issues/8627))
```

Comment:
looks to me like the incorrect issue is linked here

Reply:
```suggestion
- the `inert` attribute is now applied to outroing elements to make them invisible to assistive technology and prevent interaction. ([#8628](https://github.com/sveltejs/svelte/pull/8628))
```
indeed - thanks!